src/Decoder/SmileDecoder.php

```diff
@@ -249,7 +244,7 @@ private function decodeStringValue(int $length): string
 
         $this->context->increaseIndex($length);
 
-        return $result;
+        return iconv('UTF-8', 'ISO-8859-1', $result);
```

Comment:
iconv is missing from composer.json

Reply:
But iconv is baseline ?
https://www.php.net/manual/en/iconv.installation.php

Reply:
>although it may be disabled by compiling with --without-iconv